src/components/request/RequestForm.tsx

```diff
@@ -14,22 +14,44 @@ import {
   RequestInputButtonBlock,
   RequestFormButton,
   BondLogo,
+  ProposerAddress,
+  InputError,
 } from "./Request.styled";
 import { Duration } from "luxon";
 import calculateTimeRemaining from "helpers/calculateTimeRemaining";
 import useClient from "hooks/useOracleClient";
 import useConnection from "hooks/useConnection";
 import useReader from "hooks/useOracleReader";
+import { ChainId, CHAINS } from "constants/blockchain";
+import { ethers } from "ethers";
+import { prettyFormatNumber } from "helpers/format";
 
 const TEN_HOURS_IN_MILLSECONDS = 60 * 60 * 10 * 1000;
 const TWENTY_FOUR_HOURS_IN_MILLISECONDS = 60 * 60 * 24 * 1000;
 const RequestForm: FC = () => {
   const [currentTime, setCurrentTime] = useState(0);
   const [value, setValue] = useState("");
+  const [inputError, setInputError] = useState("");
   const inputOnChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setValue(event.target.value);
+    const v = event.target.value;
+    setValue(v);
+    setInputError("");
+    checkForInputError(v);
   };
-  const { flags, client, state } = useClient();
+  const checkForInputError = (v: string) => {
```

Comment:
we could make this a bit more portable, include decimal param: `(v:string,dec:number=18) => true` and throw the error rather than use `setInputError`.  then this could be a pure function. you just need to catch the error in `inputOnChange`

Reply:
Adjustment made.